docs/api/interfaces.md

```diff
@@ -222,6 +222,9 @@ Response to creating a new contact
 
 Before being able to use the REST Interface you will need to have set up the `CIVICRM_SITE_KEY` and the users `API_KEY`. There are three methods of creating API keys for users
 
+!!! warning
```

Comment:
````suggestion
```warning
````

Reply:
This is the syntax already used for notes, warnings and fatal errors. Why do you suggest a change?

Reply:
Eg. check just below this headline from the same page: https://docs.civicrm.org/dev/en/latest/api/interfaces/#crmapi3
Code formatting isn't fitting here.